switch-host-gcc.sh

```diff
@@ -2,6 +2,12 @@
 export SELF="$0"
 export LC_ALL=C
 
+if [ `uname` = Darwin ]; then
+    ABSPATH='realpath'
```

Comment:
Cannot we just use `realpath` unconditionally?

I see it is not available on ubuntu14 by default but it can be easily installed.  Are there any other operating systems we care about where `realpath` is not available?

Reply:
Sure, if it works on Linux, I can change it in every script.